transaction_test.go

```diff
@@ -390,6 +482,187 @@ func TestEnv(t *testing.T) {
 	}
 }
 
+func Test_StatusError(t *testing.T) {
+	t.Parallel()
+	if !CheckPamHasStartConfdir() {
+		t.Skip("this requires PAM with Conf dir support")
+	}
+
+	statuses := map[string]error{
```

Comment:
See my other comments. I think it might make sense for this to be part of the library rather than a test case. Let's discuss.